portal-kernel/src/com/liferay/expando/kernel/model/ExpandoColumnConstants.java

```diff
@@ -116,6 +116,9 @@
 	public static final String PROPERTY_DISPLAY_TYPE_GEOLOCATION =
 		"geolocation";
 
+	public static final String PROPERTY_DISPLAY_TYPE_INPUT_FIELD =
+		"input-field";
+
 	public static final String PROPERTY_DISPLAY_TYPE_NUMBER = "number";
```

Comment:
We should remove this since it's no longer needed. Don't forget to update the `getDefaultDisplayTypeProperty` method to return the proper display type.